PowerEditor/src/notepad++.exe.manifest

```diff
@@ -7,6 +7,11 @@
     name="Notepad++"
     type="win32"
 />
+<msix xmlns="urn:schemas-microsoft-com:msix.v1"
+    publisher="CN=Notepad++, O=Notepad++, L=Saint Cloud, S=Ile-de-France, C=FR"
+    packageName="NotepadPlusPlus"
+    applicationId="NotepadPlusPlus"
+/>

```

Comment:
@GurliGebis 
I think the above lines are not necessary, since I can use the context menu with v8.5 (which has been compiled without these lines). If they are really needed, please remove organization, location, province and country, as @wallrik has suggested here: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/13330#issuecomment-1466748095

Reply:
I will do that once I get close to my laptop in a few days